### PR TITLE
Backend plugins: Log deprecation warning when using the unofficial first version of backend plugins

### PR DESCRIPTION
**Pull request by marefr**

### CODE REVIEW

**Review by aknuds1 — APPROVED** [review]

**Comment by bergquist** [comment]
Could we log this on start up as well?

**Comment by marefr** [comment]
@bergquist added log warning when loading a plugin using the old plugin system. Not sure the text makes sense and/or it's enough with the "Plugin uses a deprecated version of Grafana's backend plugin system which will be removed in a future release."? Please suggest improvement

**Review by marcusolsson — APPROVED** [review]

File: pkg/plugins/backendplugin/backend_plugin.go [hunk 1]
```diff
@@ -98,6 +98,8 @@ func (p *BackendPlugin) start(ctx context.Context) error {
 			}
 		}
 	} else {
+		p.logger.Warn("Plugin uses a deprecated version of Grafana's backend plugin system which will be removed in a future release. " +
+			"Consider upgrade to a newer plugin version or reach out to the plugin repository/developer and request an upgrade. ")
```
Comment (kylebrandt) [inline pkg/plugins/backendplugin/backend_plugin.go:102]: "Consider upgrading" or "Consider an upgrade". Should there be a space after the final period? That seems odd.

**Review by bergquist — APPROVED** [review]
:+1:

File: pkg/plugins/backendplugin/backend_plugin.go (hunk 1 above)
Comment (marefr) [reply pkg/plugins/backendplugin/backend_plugin.go:102]:
```suggestion
			"Consider upgrading to a newer plugin version or reach out to the plugin repository/developer and request an upgrade.")
```

Comment (marefr) [reply pkg/plugins/backendplugin/backend_plugin.go]: I fixed it :smile: Thanks

**Review by kylebrandt — APPROVED** [review]

**Pull request merged**